### PR TITLE
fix & test: fix viewBox and add test cases of pattern

### PR DESCRIPTION
**Pull request by shinyxhh**

### CODE REVIEW

File: tester/harmony/svg/src/main/cpp/SvgGraphic.cpp [hunk 1]
```diff
@@ -291,28 +291,28 @@ void SvgGraphic::SetPatternStyle() {
...
     }
 
+    OH_Drawing_Bitmap *bitmap = OH_Drawing_BitmapCreate();
+    OH_Drawing_BitmapFormat format = {COLOR_FORMAT_RGBA_8888, ALPHA_FORMAT_OPAQUE};
+
+    OH_Drawing_BitmapBuild(bitmap, int(w), int(h), &format);
+    OH_Drawing_CanvasBind(canvas, bitmap);
+
+        // set background color to white
+    OH_Drawing_Brush *brush = OH_Drawing_BrushCreate();
```
Comment (Louis-C7) [inline tester/harmony/svg/src/main/cpp/SvgGraphic.cpp]: 用drawing::Brush

**Review by Louis-C7 — CHANGES_REQUESTED** [review]

File: tester/harmony/svg/src/main/cpp/SvgGraphic.cpp [hunk 2]
```diff
@@ -291,28 +291,28 @@ void SvgGraphic::SetPatternStyle() {
...
 
+    OH_Drawing_Bitmap *bitmap = OH_Drawing_BitmapCreate();
+    OH_Drawing_BitmapFormat format = {COLOR_FORMAT_RGBA_8888, ALPHA_FORMAT_OPAQUE};
+
+    OH_Drawing_BitmapBuild(bitmap, int(w), int(h), &format);
+    OH_Drawing_CanvasBind(canvas, bitmap);
+
+    // set background color to white
+    drawing::Brush brush;
+    OH_Drawing_BrushSetColor(brush.get(), 0xffffffff);
```
Comment (Louis-C7) [inline tester/harmony/svg/src/main/cpp/SvgGraphic.cpp]: brush.SetColor

File: tester/harmony/svg/src/main/cpp/SvgGraphic.cpp (hunk 1 above)
Comment (shinyxhh) [reply tester/harmony/svg/src/main/cpp/SvgGraphic.cpp]: 已修改

**Review by Louis-C7 — CHANGES_REQUESTED** [review]

File: tester/harmony/svg/src/main/cpp/SvgGraphic.cpp (hunk 2 above)
Comment (shinyxhh) [reply tester/harmony/svg/src/main/cpp/SvgGraphic.cpp]: 已修改

**Review by Louis-C7 — APPROVED** [review]

**Pull request merged**